Tests/XCUITests/FxScreenGraph.swift

```diff
@@ -130,6 +131,9 @@ class Action {
     static let ToggleNightMode = "ToggleNightMode"
     static let ToggleTrackingProtection = "ToggleTrackingProtection"
     static let ToggleNoImageMode = "ToggleNoImageMode"
+    
```

Comment:
❗ Swiftlint error: L134 should not have trailing spaces

Reply:
😢 It's not my first time missing such a lint error. 💭 Let me think how I can avoid this kind of error moving forward...perhaps it's the way I use XCode.

Reply:
Do you have swiftlint installed on your computer? The Xcode project is setup so that it won't run it if it's not installed (and so you wouldn't see any of the swiftlint warnings). 

Also, my trick to make sure I don't miss swiftlint warnings is by running in a terminal at the root of the project `swiftlint`, it will tell me if there's some warnings in there (since we have a bunch of build warnings in Xcode and it can be easy to miss new warnings). Some warnings can be automatically corrected by using ` swiftlint --autocorrect`, can be useful!

Reply:
yeah, that's what I was going to suggest, to run the swiftlint command locally before commiting changes

Reply:
`swiftlint --autocorrect` introduces changes in the files that I didn't edit. Better remember to specify the files when the command is invoked. 😫